src/components/settings/settings.tsx

```diff
@@ -7,12 +7,22 @@ import { useContext } from 'react';
 import { UserContext } from '../app/app-context/app-context';
 import './settings.module.css';
 
+import PageTitle from '../shared/page-title/page-title';
```

Comment:
Place new imports right about the CSS file import

Reply:
Quick question, does this just mean make the imports a single block?

Reply:
Yes, all the imports should be together in one block at the top of the file. The css file import is almost always the last import.